src/index.js

```diff
@@ -65,15 +65,15 @@ class MockHolochainConductor {
     })
   }
 
-  any (response) {
-    this.anyResponse = response
+  any (response, returnError = false) {
```

Comment:
I think I'd like the second arg in these calss to be an opts field, partly for future proofing, partly for readability in the calling code.

`mockHolochainConductor.next("error message", { returnError: true })`

is more readable than

`mockHolochainConductor.next("error message", true)`

(obv this applies to `next` and `once` as well)

Reply:
agreed